content/exception-filters.md

```diff
@@ -347,6 +347,21 @@ export class AllExceptionsFilter implements ExceptionFilter {
 }
 ```
 
+> warning **Warning** When combining an exception filter that catches everything with a filter that is bound to a specific type, the Catch-Anything filter should be declared first to allow the specific filter to correctly handle the bound type
+
+
+##### Catch-Anything Example
+
+```typescript
+// wrong
+app.useGlobalFilters(new MySpecificFilter(), new CatchAllFilter())
+// right
+app.useGlobalFilters(new CatchAllFilter(), new MySpecificFilter())
+```
+
+See #9696 for details.
```

Comment:
looks like we missed something here

Could be `See [this](https://...) for details`

Reply:
Ah, I thought it would auto-link github issues but i thought of it as markdown instead of a file that then gets turned into html. Good catch.

Reply:
updated 75c812bb